internal/testrunner/runners/policy/runner.go

```diff
@@ -98,14 +116,10 @@ func (r *runner) runTest(ctx context.Context, manager *resources.Manager, option
 	resources := resource.Resources{&policy}
 	_, testErr := manager.ApplyCtx(ctx, resources)
 	if testErr == nil {
-		// Revision 1 on creation, plus one revision for each attached policy.
-		// In some cases the revision 2 with the agent policy disappears if there is some
-		// issue with the final policy.
-		expectedRevision := 1 + len(policy.PackagePolicies)
-		if options.GenerateTestResult {
-			testErr = dumpExpectedAgentPolicy(ctx, options, testPath, policy.ID, expectedRevision)
+		if r.generateTestResult {
+			testErr = dumpExpectedAgentPolicy(ctx, r.kibanaClient, testPath, policy.ID)
 		} else {
-			testErr = assertExpectedAgentPolicy(ctx, options, testPath, policy.ID, expectedRevision)
+			testErr = assertExpectedAgentPolicy(ctx, r.kibanaClient, testPath, policy.ID)
 		}
```

Comment:
Removed `expectedRevision` as parameter since in none of those functions was used.

Should it be used that variable in those functions ? @jsoriano

Reply:
Ah no, I think the code using this variable was removed.